DataDefinitions/Vehicle.cs

```diff
@@ -10,24 +10,44 @@ public class Vehicle
         public string loadout { get; private set; }
         public int rebuilds { get; private set; }
 
+        public string vehicleDefinition
+        {
+            get => vehicleDef.edname;
+            set
+            {
+                VehicleDefinition vDef = VehicleDefinition.FromEDName(value);
+                this.vehicleDef = vDef;
+            }
+        }
+
         [JsonIgnore]
-        private VehicleDefinition definition;
+        private VehicleDefinition vehicleDef;
 
         [JsonIgnore]
-        public string localizedName => definition?.localizedName;
+        public string localizedName => vehicleDef?.localizedName;
 
         [JsonIgnore]
-        public string invariantName => definition?.invariantName;
+        public string invariantName => vehicleDef?.invariantName;
 
         [JsonIgnore]
         [Obsolete("Please be explicit and use localizedName or invariantName")]
         public string name => localizedName;
 
+        public string loadoutDescription
+        {
+            get => descriptionDef.edname;
```

Comment:
Needs null protection, as descriptionDef might be null.
`get => descriptionDef.edname;`

Reply:
Fixed.